src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -119,6 +121,11 @@ public void checkIdsAfterEach() {
     StorageTestSuite.checkForMismatchedIDs("item");
   }
 
+  @After
+  public void removeStatisticalCodes() {
+    statisticalCodeFixture.removeTestStatisticalCodes();
```

Comment:
Why is this being done after and not before?

Reply:
Should not require cleanup of statistical codes before they are created. If other test class creates them, it should be responsible for cleaning them up. If the cleanup is done before there is a chance the last test of this class will create something that makes the next test class non isolated.

Reply:
Ok. I'm biased towards resetting the state before a test starts. It means that the initial state is always cleaned up irrespective of what ran before and not resetting it after can make manual diagnostics easier (as the state is still present)

Reply:
I see. That is interesting. That puts the onus on developers to cleanup all other tests remnants while writing a test class for them to be isolated. I can see the benefit in ability to diagnose but not sure how as when test fails another will likely run and the before method will do the cleanup anyhow.

Reply:
> That puts the onus on developers to cleanup all other tests remnants while writing a test class for them to be isolated

I think of it more as preparing the state I want. I can understand framing it the other way.